### DISCUSSION TITLE
Really Struggling to Install and get going

### DISCUSSION THREAD

**Discussion by MarkCEagar** [isl-org/Open3D#3899] [Q&A]
Anychance someone can assist with getting me up and running with installation and real time streaming of point clouds ... Got some forward thinking projects to deliver yet just cant get this installed and up and running... >_<

**Comment by MeyerBuaharon** [comment]
I did something similar,
this post helped me a lot:
https://github.com/isl-org/Open3D/issues/2869#issuecomment-761942166

I would start with just copy-pasting the entire code that @guanming001 wrote down there, then I would continue from there.
